src/Components/Header.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { InputSwitch } from 'primereact/inputswitch'
+
+function Header() {
+  const theme = document.body.style
+  themeCheck()
+  function themeCheck() {
+    switch (localStorage.getItem('theme')) {
+      case 'dark':
+        theme.backgroundColor = 'black'
+        theme.color = 'white'
+        localStorage.setItem('checked', true)
+        break
+      default:
+        theme.backgroundColor = 'white'
+        theme.color = 'black'
+        localStorage.setItem('checked', false)
+    }
+  }
+  const ThemeChange = () => {
+    switch (theme.backgroundColor) {
+      case 'black':
+        theme.backgroundColor = 'white'
+        theme.color = 'black'
+        localStorage.setItem('theme', 'light')
+        localStorage.setItem('checked', false)
+        location.reload()
```

Comment:
Is the `reload` needed?

Reply:
I am using reload because I don't know any other way to rerender the Links component so that the GitHub link color gets updated whenever we switch between themes but if you know a better way please suggest.😅

Reply:
Reload is not a great user experience, it would be better if we can do it without a reload - hopefully others can suggest a way to achieve this